gui_qt/app.py

```
@@ -4474,23 +4822,58 @@ def _on_save_config(self) -> bool:
             return False
 
 
+    def _sync_state_game_root_from_settings(self, value: object) -> None:
+        if not isinstance(value, str) or not value.strip():
+            return
+        try:
+            normalized, _adjusted = self.state.normalize_game_root(value)
+            self.state._game_root = normalized
+            self.state._game_root_redirect_from = None
+            self._refresh_project_label()
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)